pages/about.tsx

```diff
@@ -40,75 +40,87 @@ const About = () => {
+          <p className="w-full md:w-2/3">
+            chasing your goals and working hard for it is important, getting hobbies and taking a
             break once in a while isnt a bad thing either.
           </p>
-          <p className="w-2/3">
+          <p className="w-full md:w-2/3">
             When I&apos;m free, I mostly spend my time helping my mom with errands, play video
             games. and when i get the chance, i go swimming or hang out with friends.
           </p>
-          <p className="w-2/3">
+          <p className="w-full md:w-2/3">
             all of these help to calm me down, relieve a bit of that stress, and focus on what to
             work on next. taking breaks isn&apos;t a bad thing when it doesn&apos;t pull you away
             from what is important to you.
           </p>
           <motion.p
             style={{ y: yLabel2 }}
-            className="absolute right-[5%] text-black/10 text-[100px] font-extrabold"
+            className="absolute text-end right-[5%] text-black/10 text-[100px] font-extrabold"
           >
             My Hobbies
           </motion.p>
         </div>
-        <div className="relative w-1/2">
-          <motion.div className="absolute top-0 left-0" style={{ y: yImage1 }}>
-            <Image src="/images/helping.jpg" width={300} height={400} />
+        <div className="hidden md:flex relative w-1/2">
+          <motion.div
+            whileInView={{ opacity: [0, 1] }}
+            transition={{ duration: 0.5 }}
+            className="absolute top-0 left-0 w-[180px] h-[250px] lg:w-[200px] lg:h-[300px] 2xl:w-[300px] 2xl:h-[400px]"
+            style={{ y: yImage1 }}
+          >
+            <Image src="/images/helping.jpg" layout="fill" />
           </motion.div>
           <motion.div
-            className="absolute z-10 top-0 left-0 mt-[5%] ml-[25%]"
+            whileInView={{ opacity: [0, 1] }}
+            transition={{ duration: 0.5, delay: 0.5 }}
+            className="absolute z-10 top-0 left-0  2xl:mt-[5%] ml-[25%] w-[180px] h-[250px] lg:w-[200px] lg:h-[300px] 2xl:w-[300px] 2xl:h-[400px]"
             style={{ y: yImage2 }}
           >
-            <Image src="/images/game.jpg" width={300} height={400} />
+            <Image src="/images/game.jpg" layout="fill" />
           </motion.div>
-          <motion.div className="absolute top-0 left-0 mt-[20%] ml-[50%]" style={{ y: yImage3 }}>
-            <Image src="/images/swimming.jpg" width={300} height={400} />
+          <motion.div
+            whileInView={{ opacity: [0, 1] }}
+            transition={{ duration: 0.5, delay: 1 }}
+            className="absolute top-0 left-0 2xl:mt-[20%] ml-[50%] w-[180px] h-[250px] lg:w-[200px] lg:h-[300px] 2xl:w-[300px] 2xl:h-[400px]"
+            style={{ y: yImage3 }}
+          >
+            <Image src="/images/swimming.jpg" layout="fill" />
           </motion.div>
```

Comment:
These look almost similar, with a little difference in class names & transitions. Create a small object with the image field and distinctive class name properties & render the div. It will look cleaner.